vector_index.go

```diff
@@ -29,6 +29,12 @@ func init() {
 	reflectStaticSizeVectorDoc = int(reflect.TypeOf(vd).Size())
 }
 
+type VectorIndex interface {
+	Search([]float32, int64) ([]float32, []int64, error)
+
+	Close()
```

Comment:
`Close()` need not return an error?

Reply:
Going by this I think not - https://github.com/blevesearch/zapx/blob/322e3d773a334be3b00d3661b88eb3cc3bb3675b/faiss_vector_posting.go#L344